OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -43,21 +43,33 @@ public class MinelayerInfo : ITraitInfo, Requires<RearmableInfo>
 		[Desc("Sprite overlay to use for minefield cells hidden behind fog or shroud.")]
 		public readonly string TileUnknownName = "build-unknown";
 
+		[Desc("Only allow laying mines on listed terrain types. Leave empty to allow all terrain types.")]
+		public readonly HashSet<string> TerrainTypes = new HashSet<string>();
+
+		[Desc("Cursor to display when able to deploy (lay mine).")]
```

Comment:
Having `lay mine` in parenthesis looks a bit odd to me. Imho we should just use `Cursor to display when able to lay a mine.` here and below (with `unable` there obviously).

Reply:
Changed.